client/my-sites/post-type-filter/index.jsx

```diff
@@ -180,7 +182,11 @@ export class PostTypeFilter extends Component {
 							initialValue={ query.search }
 							isOpen={ this.props.getSearchOpen() }
 							onSearch={ this.props.doSearch }
-							placeholder={ `${ searchPagesPlaceholder }…` }
+							placeholder={
+								searchPagesPlaceholder
+									? `${ searchPagesPlaceholder }…`
+									: this.props.translate( 'Search…', { context: 'verb: imperative' } )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **57** times:
`translate( 'Search…' )` *ES Score: 9*


:information_source: This string already exists without a **context**. Only add a context if the meaning of the string is very specific.